ififuncs.py

```diff
@@ -735,6 +735,12 @@ def get_user():
     elif user == '7':
         user = 'Eoin O\'Donohoe'
         time.sleep(1)
+    elif user == '8':
+        user = 'Yujing Haung'
```

Comment:
typo, should be Huang ;)

Reply:
cool, thanks for catching that.
how do i change it? i know i should know, - sorry.

Reply:
can't remember off the top of my head, you either update directly in your patch-6 here https://github.com/raecasey/IFIscripts/blob/patch-6/ififuncs.py#L739 or else you do it via this PR.
If stuck, I can come over and take a look? https://github.com/kieranjol/IFIscripts/pull/337/files